src/ui-test/draft-manifest-files.test.ts

```diff
@@ -0,0 +1,367 @@
+import * as assert from 'assert';
+import * as fs from 'fs';
+import path = require('path');
+import {
+   TitleBar,
+   VSBrowser,
+   InputBox,
+   ActivityBar,
+   Workbench,
+   Key,
+   until,
+   QuickPickItem,
+   By
+} from 'vscode-extension-tester';
+import {setWorkspace} from './common';
+describe('Draft Kubernetes Deployment Test', () => {
+   const pathToWorkspace =
+      path.resolve(__dirname, '../../src/ui-test/test-repo/flask-hello-world') +
+      '/';
+   let browser: VSBrowser;
+
+   before(async function () {
+      this.timeout(100000);
+      browser = VSBrowser.instance;
+      await setWorkspace(pathToWorkspace, browser);
+   });
+
+   it('drafts manifest files', async function () {
+      this.timeout(200000);
+      //open command palette
+      let prompt = await new Workbench().openCommandPrompt();
+      const picks = await prompt.getQuickPicks();
+      assert.notStrictEqual(picks.length, 0);
+
+      //select quick pick to draft manifest files
+      await prompt.selectQuickPick(
+         'AKS Developer: Draft a Kubernetes Deployment and Service'
+      );
+      await browser.driver.wait(
+         until.elementIsVisible(
+            await browser.driver.findElement(
+               By.xpath(".//input[@aria-describedby='quickInput_message']")
+            )
+         )
+      );
+      await prompt.confirm();
+      await browser.driver.wait(
+         until.elementLocated(By.className('quick-input-list'))
+      );
+      await prompt.selectQuickPick('Manifests');
+      await browser.driver.wait(
+         until.elementLocated(
+            By.xpath(".//input[@aria-describedby='quickInput_message']")
+         )
+      );
+      await prompt.setText('flask-hello-world');
+      await prompt.confirm();
+      await browser.driver.wait(
+         until.elementTextContains(
```

Comment:
So why couldnt this be leveraged for the port text set on like 69?

Reply:
I tried that, but it wasn't working. For some reason, setting the port as '8080' is taking more time, so if I confirm it without using sleep, it's trying to confirm on empty input.

Reply:
But isnt that what the wait is for? wait for input to change?

Reply:
updated the code to use locators instead.